### PR TITLE
First approach for i18n for new Editor

### LINKED ISSUE #6464: Add i18n proposal
Hello [polyglot.js](http://airbnb.io/polyglot.js/).

Add only english as a first version.

### PR DESCRIPTION
**Pull request by xavijam**
It comes from @javisantana's proposal, and try to fix #6464.

Maps!:

screen shot 2016-01-27 at 16 49 31

screen shot 2016-01-27 at 16 49 39

How does it sound @javisantana @viddo @javierarce @acanimal @alonsogarciapablo?

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/editor.js [hunk 1]
```diff
@@ -1,5 +1,17 @@
+var $ = require('jquery');
 var _ = require('underscore');
 var cdb = require('cartodb-deep-insights.js');
+var Polyglot = require('node-polyglot');
+var Locale = require('../../locale/index');
+var ACTIVE_LOCALE = 'en';
+var EditorTemplate = require('./editor.tpl'); // TODO: remove it
+
+window.polyglot = new Polyglot({
```
Comment (javisantana) [inline lib/assets/javascripts/cartodb3/editor.js]: this should be under cdb scope IMO

File: lib/assets/javascripts/cartodb3/editor.tpl [hunk 2]
```diff
@@ -0,0 +1 @@
+<div><%- polyglot.t('dashboard.hello') %></div>
```
Comment (javisantana) [inline lib/assets/javascripts/cartodb3/editor.tpl]: We should use the cartodb object and not polyglt directly

File: lib/assets/javascripts/cartodb3/editor.js (hunk 1 above)
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor.js]: I'm not against that, but we should set the scope of this translations, should we use them within CartoDB.js and Deep-insights?

Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor.js]: Or should they be ready for current Editor?

**Comment by viddo** [comment]
Looks OK to me :+1:

**Comment by acanimal** [comment]
Fine for me

**Comment by javisantana** [comment]
:es:

**Comment by xavijam** [comment]
Added example testing pluralize. There is a key variable for the count, ```smart_count```.

**Comment by javierarce** [comment]
Go(ooo)od!

**Comment by xavijam** [comment]
Merging into "setup frontend v3 editor" branch.

**Comment by alonsogarciapablo** [comment]
`<%- cdb.T('fun.times.ahead') %>`

**Comment by xavijam** [comment]
:joy:

**Pull request merged**